test/resources/logback.xml

```diff
@@ -6,7 +6,7 @@
             </pattern>
         </encoder>
     </appender>
-    <root level="ERROR">
+    <root level="INFO">
```

Comment:
can we keep the previous level here? otherwise the terminal is flooded with the logs making it hard to read

Reply:
I changed this because it said there was no test coverage for lines of code with WARN and INFO level logs. This lowered the test coverage overall, making it below the minimum %

Reply:
I can change it to WARN level?

Reply:
changed it back to ERROR 👍🏾